index.html

```diff
@@ -1,13 +1,132 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="zxx">
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css"/>
-    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
+    <link rel="icon" href="./img/favicon.ico" type="image/x-icon">
+    <title>The April cultural festival in Tarija</title>
 </head>
 <body>
+    <header>
+        <div class="mediabar"><a href="https://www.facebook.com/" target="_blank" rel="noreferrer noopener" aria-label="this is an external link (opens in a new tab)"><img src="./img/facebook.png" alt="facebook"></a><a href="https://twitter.com" target="_blank" rel="noreferrer noopener" aria-label="this is an external link (opens in a new tab)"><img src="./img/twitter.png" alt="twitter"></a><a href=" https://alondra87.github.io/Portfolio1/" target="_blank" rel="noreferrer noopener" aria-label="this is an external link (opens in a new tab)">My page </a><a href=""> English </a><a href="https://www.google.com/" target="_blank" rel="noreferrer noopener" aria-label="this is an external link (opens in a new tab)"> Log out </a></div>
+        <nav class="navbar">
+            <a href="#"><img src="./img/logo.jpg" alt="logo" class="logo"></a>
+            <ul class="nav-menu">
+                <li class="nav-item">
+                    <a href="about.html" class="nav-link">About</a> 
+                </li>
+                <li class="nav-item">
+                    <a href="#program" class="nav-link">Program</a> 
+                </li>
+                <li class="nav-item">
+                    <a href="#speakers" class="nav-link">Speakers</a> 
+                </li>
+                <li class="nav-item">
+                    <a href="#sponsor" class="nav-link">Sponsors</a> 
+                </li>
+                <li class="nav-item">
+                    <a href="https://alondra87.github.io/Capstone/about.html#memories" class="nav-link">Memories</a> 
+                </li>
+                <li class="nav-item">
+                    <a href="https://www.casadelaculturatarija.com/" class="nav-linkAT">AT&nbsp;Campaign</a> 
+                </li>
+            </ul>
+            <div class="hamburger">
```

Comment:
- [ ] [Optional] The nav links should align to the right of the navbar, please consider moving the links to the right and reducing some margin/padding

|Your design|CC design|
|---|---|
|![Screenshot_20220519_211814](https://user-images.githubusercontent.com/25227022/169398471-f85caf8f-1330-4908-ba2d-17965252e16e.png)|![Screenshot_20220519_211835](https://user-images.githubusercontent.com/25227022/169397631-ada4b8d2-d784-4b33-bafd-dae4a3c459aa.png)|